functional_tests.go

```diff
@@ -5495,13 +5497,6 @@ func testPresignedPostPolicy() {
 		return
 	}
 
-	// Save the data
-	_, err = c.PutObject(context.Background(), bucketName, objectName, bytes.NewReader(buf), int64(len(buf)), minio.PutObjectOptions{ContentType: "binary/octet-stream"})
-	if err != nil {
-		logError(testName, function, args, startTime, "", "PutObject failed", err)
-		return
-	}
-
```

Comment:
I think this is just a copy paste mistake, not sure why we were doing a PutObject here, as we are running the PostPolicy presigned request via the http client further down.